sdcm/utils/csrangehistogram.py

```diff
@@ -139,7 +137,7 @@ def build_histogram_from_file(hdr_file: str,
 
     @staticmethod
     def build_histograms_range_with_interval(path: str,
-                                             start_time=0, end_time=sys.maxsize,
+                                             start_time: int | float, end_time: int | float,
```

Comment:
you dropped here the defaults

Reply:
Yes. i have to do that to avoid infinite empty histogram building